### PR TITLE
fix undefined description attribute error

### PR DESCRIPTION
**Pull request by danpaul**
This fixed a fatal undefined variable error I was getting.

### CODE REVIEW

File: src/Elements/HrefVariablesElement.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ public function getAllVariables()
             $memberContent = $member->getContent();
 
             $hrefVariable = new HrefVariable();
-            $hrefVariable->description = $member->getMetaData()['description'];
+            $hrefVariable->description = empty($member->getMetaData()['description']) ? '' : $member->getMetaData()['description'];
```
Comment (hendrikmaus) [inline src/Elements/HrefVariablesElement.php]: Please use PHP7's ` ?? ''` instead of `empty` and the ternary operator. Like, for example, in PR #9

**Review by hendrikmaus — CHANGES_REQUESTED** [review]

File: src/Elements/HrefVariablesElement.php (hunk 1 above)
Comment (danpaul) [reply src/Elements/HrefVariablesElement.php]: good call, updated

**Comment by hendrikmaus** [comment]
Sry, closed by accident and reopened. Will merge and release.

**Comment by hendrikmaus** [comment]
Released as v0.1.6, thanks for the contribution.

**Pull request merged**